beta-src/src/interfaces/UserData.ts

```diff
@@ -0,0 +1,12 @@
+import Vote from "../types/Vote";
+import { CountryTableData } from "./CountryTableData";
+
+export interface UserData {
+  /**
+   * TODO
+   * May need additional data
+   */
+  id: string;
+  countryTableData: CountryTableData;
+  votes: Vote;
+}
```

Comment:
Not following what this file is for?

Reply:
We still need to show current user's votes even if game doesn't show vote status of players.
Wasn't sure how we are going to have countryTableData stored in the redux store but assumed we will treat current user's data separately.

Reply:
How do we know which country in CountryTableData belongs to current user?

Reply:
I think we should use something like a `countryID`, since that is what is used when fetching from the API.

Reply:
@bryancodazen is this good to go? I think later we can shape the userData structure.